application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -3738,6 +3744,15 @@ enum StopAlertType {
   TRIPS
 }
 
+enum StopType {
+  "An area or zone represented by a polygon. Introduced by the GTFS-Flex spec process."
+  LOCATION
+  "A group of stops. Introduced by the GTFS-Flex spec process."
+  LOCATION_GROUP
```

Comment:
Are the locations always regular stops or can they be geojson locations? I had a feeling that they might be mostly geojson locations, but I couldn't make sense by reading the docs for a minute :D.

Reply:
Not sure, maybe using the GTFS terminology here is fine.